Tests/Functional/Asynchronous/Expectations/main.swift

```diff
@@ -551,7 +551,37 @@ class ExpectationsTestCase: XCTestCase {
         RunLoop.main.run(until: Date() + 1)
     }
 
-    static var allTests = {
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor' failed \(\d+\.\d+ seconds\)
+    func test_waitForExpectationsAsync() async {
```

Comment:
I think the test function name in these `// CHECK` lines doesn't match the actual method name, which will likely cause CI to fail

Reply:
Good eye, thanks.